docs/application/dotnet/guides/applications/uiapplication/ui-app.md

```diff
@@ -13,10 +15,10 @@ To create a basic UI application, you must:
 
 - Manage [application states and transitions](#state_trans) during the
```

Comment:
Line 16: "Manage [application states and transitions]..." needs to be updated, the hyperlink is leading to Application States and Transitions section with anchor #state_trans, but as the section was removed, it is showing the wrong link.
There are two solutions you can incorporate:
1. Just deleting line 16~17
2. Update this link to go to application_lifecycle.md, instead of inserting line 3? This sentence wants to give a link to "Application states & transitions" anyway, which are in application_lifecycle.md now.